server.js

```diff
@@ -15,6 +15,13 @@ app.use(express.static('public'))
 app.post('/audit', async (req, res) => {
   const branch = req.body.text
 
+  if (!branch.match(/[0-9]-[0-9]-x/)) {
```

Comment:
Just so we don't cry in a year when we get to `10`
```suggestion
  if (!branch.match(/[0-9]+-[0-9]+-x/)) {
```